docs/howto/jupyterhub.md

```diff
@@ -0,0 +1,75 @@
+## Log in to Wynton JupyterHub (JHUB)
+
+As of now JHUB can be accessed either by connecting to VPN or from campus network using [JupyterHub Login](https://jhub.wynton.ucsf.edu/hub/login).
+
+JHUB login is confined only to non-PHI Wynton user accounts. If you requested us to convert your account to a PHI account or setup your account as a PHI account, then you won't be able to login into JHUB.
+
+Please setup your own environment before getting started. The base environment on JupyterHub is installed locally on the server and won't be accessible from the cluster. 
+
+There are several ways one can setup an environment, we recommend using `virtualenv`. See [Installing packages to a virtual environment](https://wynton.ucsf.edu/hpc/howto/python.html#2-installing-to-a-virtual-environment-aka-virtualenv) for instructions.
+
+## Creating a kernel for virtual environment
+
+To create a kernel that you can access from JHUB follow the steps below
+Activate the virtual environment:
+
+```python
+[alice@{{ site.devel.name }} ~]$ cd my_project
+[alice@{{ site.devel.name }} my_project]$ . bin/activate
+(my_project) [alice@{{ site.devel.name }} my_project]$ 
+```
+
+Install "**ipykernel**" package
+
+```bash
+(my_project) [alice@{{ site.devel.name }} my_project]$ python -m ipykernel install --user --name=<env_name>
```

Comment:
1. Per <https://ipython.readthedocs.io/en/stable/install/index.html>, I think we're missing instructions how to install the **ipykernel** module, i.e.

```sh
(my_project) [alice@{{ site.devel.name }} my_project]$ python -m pip install ipykernel
```

2. Can `--name=...` be dropped?  Or, can a more informative example name be used, e.g. `--name=my_kernelspec`, so it's not mistake for a Python virtual environment name?

Reply:
Regarding 1. Oh, maybe, you've already installed **ipykernel** globally in the **base** environment on that server?